.github/workflows/tests_lgpu_python.yml

```diff
@@ -235,11 +235,13 @@ jobs:
           mv coverage.xml coverage-${{ github.job }}-${{ matrix.pl_backend }}.xml
 
       - name: Upload code coverage results
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: ubuntu-codecov-results-python
           path: ./main/coverage-${{ github.job }}-${{ matrix.pl_backend }}.xml
```

Comment:
Is there no conflict between backends here?

Reply:
We have a single Backend here. I will expand the name, though.